woocommerce-gateway-genoapay/includes/class-genoapay.php

```diff
@@ -50,18 +50,23 @@ private static function init_hooks() {
 		public static function woocommerce_template_genoapay_details() {
 			global $product;
 			$genoapay_gateway = new WooCommerce_Gateway_Genoapay();
+            $minimum_amount = $genoapay_gateway->getMinimumAmount();
```

Comment:
indenting is funny?

Reply:
should be fixed now, was just my IDE settings for indents for PHP, should be converted from spaces to tabs now